src/cron.ts

```diff
@@ -64,14 +65,20 @@ export const getAllRFCRemarks = async (
 
     logger.info(`Available referendas: ${query}`);
     const ongoing: OpenReferenda[] = [];
-    const completed: CompletedReferenda[] = [];
+    const finished: FinishedReferenda[] = [];
 
     for (const index of Array.from(Array(query).keys())) {
       logger.info(`Fetching elements ${index + 1}/${query}`);
 
+      type Finished = [number, { who: string; amount: number } | null, unknown | null];
```

Comment:
We don't care about those values, other than the first number, right?

Then maybe it would be more readable this way, to communicate that we're dropping the rest:

```
type Finished = [number, ...unknown[]]
```